showcase/src/app/welcome/pages/home/_home-page.component.scss

```diff
@@ -1,94 +0,0 @@
-//set the header image
```

Comment:
Sorry, I didn't mean to replace this file with a `xxx-theme.scss` one but to move all the "themable" properties to the `xxx-theme.scss` file and leave in this one the rest